src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,82 @@
+.watch {
+  margin: 50px auto;
+  @include square ($watch-width);
+  background-color: #aebede;
+  border-radius: 50%;
+  border: 6px solid #414141;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+
+
+   &__minute-hand {
+    height: $watch-width * 0.4;
+    width: $watch-width * 0.035;
+    background-color: rgb(127, 95, 135);
+    position: absolute;
+    bottom: 50%;
+    border-radius: 50%;
+
+    animation: move 3600s steps(60, end) infinite;
+    transform-origin: center bottom;
+
+    @keyframes move {
+
+      to {
+        transform: rotate(360deg);
+      }
+    }
+  }
+
+  &__second-hand{
+    height: $watch-width * 0.28;
+    width: $watch-width * 0.025;
+    background-color: rgb(140, 57, 108);
+    position: absolute;
+    bottom: 50%;
+    border-radius: 50%;
+
+    animation: move 60s steps(60, end) infinite;
+    transform-origin: center bottom;
+
+    @keyframes move {
+
+      to {
+        transform: rotate(360deg);
+      }
+    }
+
+  }
```

Comment:
```suggestion
```

Reply:
I thought I have to set the animation for each object separately....ty for tip, i I wasnt attentive((